### PR TITLE
chore: updating GPU build.

### PR DESCRIPTION
**Pull request by drugilsberg**
Minor improvements in RT training pipeline.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,9 +9,9 @@ joblib>=1.1.0
 gt4sd-molformer>=0.1.1
 keras>=2.3.1,<2.11.0
 keybert>=0.7.0
+markdown-it-py<3.0.0,>=2.2.0
```
Comment (jannisborn) [inline requirements.txt:12]: If this is a requirement of the core package it should be added to the `setup.cfg` too, in order to keep things in sync as much as possible

**Review by jannisborn — COMMENTED** [review]
Looks good, did you verify the new installation also locally (on Mac)?

File: requirements.txt (hunk 1 above)
Comment (drugilsberg) [reply requirements.txt:12]: it's not a direct requirement of core hence the decision to leave it out, it's a dependency that gets installed with a messy bound in conda builds.

**Comment by jannisborn** [comment]
Since the installation for CPU also changed mildly, I think we should verify that it works on Mac. I'm doing this now, will approve the PR once my local tests pass

**Comment by jannisborn** [comment]
tests pass locally

**Review by jannisborn — APPROVED** [review]

**Pull request merged**